src/test/java/com/crowdin/client/bundles/BundlesApiTest.java

```diff
@@ -109,4 +115,31 @@ public void listFileCollectionResourceTest() {
         assertEquals(fileCollectionResourceResponse.getName(), bundleResourceName);
     }
 
+    @Test
+    public void downloadBundleTest() {
+        ResponseObject<DownloadLink> response = this.getBundlesApi().downloadBundle(projectId, bundleId, exportId);
+        assertEquals("test.com", response.getData().getUrl());
+    }
+
+    @Test
+    public void exportBundleTest() {
+        Bundle request = new Bundle();
+        request.setName(name);
+        request.setFormat(format);
+        request.setSourcePatterns(Collections.singletonList("/master/"));
+        request.setIgnorePatterns(Collections.singletonList("/master/environments/"));
+        request.setExportPattern(pattern);
+        request.setLabelIds(Collections.singletonList(0L));
+
+        ResponseObject<BundleExport> response = this.getBundlesApi().exportBundle(projectId, bundleId, request);
+        assertEquals(exportId, response.getData().getIdentifier());
+        assertEquals(2,response.getData().getAttributes().getBundleId());
+    }
+
+    @Test
+    public void exportBundleStatusTest() {
```

Comment:
the name of the test and tested method should also be changed according to the previous comment